src/boards/arcade_feather_nrf52840_express/board.h

```diff
@@ -0,0 +1,143 @@
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#ifndef _FEATHER_NRF52840_H
+#define _FEATHER_NRF52840_H
+
+#define _PINNUM(port, pin)    ((port)*32 + (pin))
+
+/*------------------------------------------------------------------*/
+/* LED
+ *------------------------------------------------------------------*/
+#define LEDS_NUMBER           2
+#define LED_PRIMARY_PIN       _PINNUM(1, 15)
+#define LED_SECONDARY_PIN     _PINNUM(1, 10)
+#define LED_STATE_ON          1
+
+#define LED_NEOPIXEL           _PINNUM(0, 16)
+#define NEOPIXELS_NUMBER      1
+#define BOARD_RGB_BRIGHTNESS  0x040404
+
+/*------------------------------------------------------------------*/
+/* BUTTON
+ *------------------------------------------------------------------*/
+#define BUTTONS_NUMBER        2
+#define BUTTON_1              _PINNUM(1, 02)
+#define BUTTON_2              _PINNUM(0, 10)
+#define BUTTON_PULL           NRF_GPIO_PIN_PULLUP
+
+/*------------------------------------------------------------------*/
+/* UART (only used by nRF52832)
+ *------------------------------------------------------------------*/
+#define RX_PIN_NUMBER         8
+#define TX_PIN_NUMBER         6
+#define CTS_PIN_NUMBER        0
+#define RTS_PIN_NUMBER        0
+#define HWFC                  false
+
+//--------------------------------------------------------------------+
+// BLE OTA
+//--------------------------------------------------------------------+
+#define BLEDIS_MANUFACTURER   "Adafruit Industries"
+#define BLEDIS_MODEL          "Feather nRF52840 Express"
+
+//--------------------------------------------------------------------+
+// USB
+//--------------------------------------------------------------------+
+#define USB_DESC_VID           0x239A
+#define USB_DESC_UF2_PID       0x0029
+#define USB_DESC_CDC_ONLY_PID  0x002A
+
+//------------- UF2 -------------//
+#define UF2_PRODUCT_NAME      "Adafruit Feather nRF52840 Express"
+#define UF2_VOLUME_LABEL      "ARCADE-N4"
+#define UF2_BOARD_ID          "nRF52840-Feather-revD"
+#define UF2_INDEX_URL         "https://www.adafruit.com/product/4062"
+
+#ifdef DEFINE_BOOTLOADER_CONFIG
+#define BOOTLOADER_CONFIG_PRESENT 1
+__attribute__((used, section(".bootloaderConfig")))
+const uint32_t bootloaderConfig[] = {

```

Comment:
Thanks, I understand the great idea behind CF2 now :) . Though 1KB is only 128 (minus 1) pairs of config. It is a bit small IMHO.  Should we go to 2KB for 512 pairs. The current head room for bootloader is still plenty for expanding. 

Total size is 40KB, current compiled is 30KB, 2 KB for CF2, and 8KB left is good enough

Reply:
sounds good, will do!